src/SQLite.cs

```diff
@@ -289,7 +289,9 @@ public SQLiteConnection (string databasePath, SQLiteOpenFlags openFlags, bool st
 			StoreDateTimeAsTicks = storeDateTimeAsTicks;
 
 			BusyTimeout = TimeSpan.FromSeconds (0.1);
-
+			if (openFlags.HasFlag (SQLiteOpenFlags.ReadWrite)) {
+				ExecuteScalar<string> ("PRAGMA journal_mode=WAL");
```

Comment:
I believe this should just be `Execute()` since you're not looking at the return value that `ExecuteScalar` provides.

Reply:
@Clancey This seems to break compatibility with sqlcipher as the decryption PRAGMA is executed after this PRAGMA. This one fails with the message that the database is encrypted.